libs/SmartSync/SmartSync/Classes/Manager/SFSmartSyncSyncManager.m

```diff
@@ -447,6 +457,7 @@ - (void) cleanResyncGhosts:(NSNumber*)syncId completionStatusBlock:(SFSyncSyncMa
         [self log:SFLogLevelError format:@"Cannot run cleanResyncGhosts:%@:wrong type:%@", syncId, [SFSyncState syncTypeToString:sync.type]];
         return;
     }
+    [self logAnalyticsEventWithSyncState:sync name:@"cleanResyncGhosts" numRecords:nil];
```

Comment:
For consistency, could we move the event creation to when the cleanResyncGhosts complete.
Then we could report back the number of records got actually cleaned up.